### ISSUE TITLE
Filter-control and Cookie do not work for the first time

### ISSUE THREAD

**Issue by SEModer** [wenzhixin/bootstrap-table#4158]
After the first sets of the value in the cookie(column filter), the search stops working, and indeed, any appeal to the server. After refresh, all good, and work fine. Only after first set value to cookie this problem manifests itself.
For fix it, i'm comment lines on bootstrap-table-cookie.js:
[ from 352 to 354 lines](https://github.com/wenzhixin/bootstrap-table/blob/078a0e3ca279e48f226cf9cac1e0e5f614eeccd0/dist/extensions/cookie/bootstrap-table-cookie.js#L352-L354)

[CodePen](https://codepen.io/anon/pen/vbprea)

**Comment by SEModer** [comment]
No idea? or it's only on me))?

**Comment by pchaganti** [comment]
The addrbar doesnt work for search either when the cookie extension is enabled.

**Comment by UtechtDustin** [comment]
Fixed in the newest version, demo: https://live.bootstrap-table.com/code/UtechtDustin/39.

**Comment by SEModer** [comment]
Nothing fixed...

**Comment by SEModer** [comment]
> Fixed in the newest version, demo: https://live.bootstrap-table.com/code/UtechtDustin/39.

May be you have new fix for this issue, when sidePagination="server"? or comment this line https://github.com/wenzhixin/bootstrap-table/blob/42b2e388d6472c87e281b60248481be7e8ccb5f0/dist/extensions/cookie/bootstrap-table-cookie.js#L1729 is a solution for this problem =)?

**Comment by crochut** [comment]
> Fixed in the newest version, demo: https://live.bootstrap-table.com/code/UtechtDustin/39.

Not fixed, add data-search="true" on your table in your sample, then type 0 on the first column search field. 
It will filter and display 3 lines. 
Then erase your "0" with backspace (the field is empty), but no refresh is done. You still have 3 lines.

**Comment by crochut** [comment]
> > Fixed in the newest version, demo: https://live.bootstrap-table.com/code/UtechtDustin/39.
> 
> May be you have new fix for this issue, when sidePagination="server"? or comment this line
> 
> https://github.com/wenzhixin/bootstrap-table/blob/42b2e388d6472c87e281b60248481be7e8ccb5f0/dist/extensions/cookie/bootstrap-table-cookie.js#L1729
> 
> is a solution for problem =)?

Yes delete this line works for me !

**Comment by UtechtDustin** [comment]
duplicate of #4578
